### ISSUE TITLE
pip update error from 0.30.18 to 0.30.20 

### ISSUE THREAD

**Issue by pur80a** [brentp/cyvcf2#271]
```
$  python3 --version # same for python 3.7, 3.9, 3.10 and 3.11 
Python 3.8.16
$ pip3 freeze --all | sed 's/==/>=/' > tmp.file # q
$ pip3 install -r tmp.file --upgrade
... 
Building wheels for collected packages: cyvcf2
  Building wheel for cyvcf2 (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for cyvcf2 (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [102 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-cpython-38
      creating build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/__init__.py -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/__main__.py -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/cli.py -> build/lib.linux-x86_64-cpython-38/cyvcf2
      creating build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/__init__.py -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test_cli.py -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test_hemi.py -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test_reader.py -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test_writer.py -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      running egg_info
      writing cyvcf2.egg-info/PKG-INFO
      writing dependency_links to cyvcf2.egg-info/dependency_links.txt
      writing entry points to cyvcf2.egg-info/entry_points.txt
      writing requirements to cyvcf2.egg-info/requires.txt
      writing top-level names to cyvcf2.egg-info/top_level.txt
      reading manifest file 'cyvcf2.egg-info/SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      adding license file 'LICENSE'
      writing manifest file 'cyvcf2.egg-info/SOURCES.txt'
      copying cyvcf2/cyvcf2.c -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/cyvcf2.pxd -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/cyvcf2.pyx -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/helpers.c -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/helpers.h -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/relatedness.h -> build/lib.linux-x86_64-cpython-38/cyvcf2
      copying cyvcf2/tests/bug.vcf.gz -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/decomposed.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/empty.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/issue_198.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/issue_44.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/no-seq-len.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/no-seq-names.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/o.vcf.gz -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/seg.vcf.gz -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-alt-repr.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-diff.csi -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-format-string.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-genotypes.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-haploidX.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-hemi.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-invalid-header.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-multiallelic-homozygous-alt.vcf.gz -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-multiallelic-homozygous-alt.vcf.gz.tbi -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-no-genotypes.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-strict-gt-option-flag.vcf.gz -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test-strict-gt-option-flag.vcf.gz.tbi -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.bug.117.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.comp_het.3.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.isa.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.mnp.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.snpeff.bcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.snpeff.bcf.csi -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.snpeff.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.vcf.gz -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test.vcf.gz.tbi -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test_gt_alt_freqs.vcf -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      copying cyvcf2/tests/test_gt_bases.vcf.gz -> build/lib.linux-x86_64-cpython-38/cyvcf2/tests
      running build_ext
      building 'cyvcf2.cyvcf2' extension
      creating build/temp.linux-x86_64-cpython-38
      creating build/temp.linux-x86_64-cpython-38/cyvcf2
      creating build/temp.linux-x86_64-cpython-38/htslib
      creating build/temp.linux-x86_64-cpython-38/htslib/cram
      creating build/temp.linux-x86_64-cpython-38/htslib/htscodecs
      creating build/temp.linux-x86_64-cpython-38/htslib/htscodecs/htscodecs
      gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Ihtslib -Icyvcf2 -I/tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include -I/home/apps/python3/3.8.16/include/python3.8 -c cyvcf2/cyvcf2.c -o build/temp.linux-x86_64-cpython-38/cyvcf2/cyvcf2.o -Wno
-sign-compare -Wno-unused-function -Wno-strict-prototypes -Wno-unused-result -Wno-discarded-qualifiers
      In file included from /tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/ndarraytypes.h:1830,
                       from /tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                       from /tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                       from cyvcf2/cyvcf2.c:842:
      /tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
       #warning "Using deprecated NumPy API, disable it with " \
        ^~~~~~~
      cyvcf2/cyvcf2.c: In function ‘__pyx_pf_6cyvcf2_6cyvcf2_3VCF_8seqnames___get__’:
      cyvcf2/cyvcf2.c:14573:18: warning: assignment to ‘char **’ from incompatible pointer type ‘const char **’ [-Wincompatible-pointer-types]
         __pyx_v_cnames = bcf_hdr_seqnames(__pyx_v_self->hdr, (&__pyx_v_n));
                        ^
      cyvcf2/cyvcf2.c:14682:22: warning: assignment to ‘char **’ from incompatible pointer type ‘const char **’ [-Wincompatible-pointer-types]
             __pyx_v_cnames = bcf_index_seqnames(__pyx_v_self->hidx, __pyx_v_self->hdr, (&__pyx_v_n));
                            ^
      cyvcf2/cyvcf2.c:14765:22: warning: assignment to ‘char **’ from incompatible pointer type ‘const char **’ [-Wincompatible-pointer-types]
             __pyx_v_cnames = tbx_seqnames(__pyx_v_self->idx, (&__pyx_v_n));
                            ^
      gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Ihtslib -Icyvcf2 -I/tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include -I/home/apps/python3/3.8.16/include/python3.8 -c cyvcf2/helpers.c -o build/temp.linux-x86_64-cpython-38/cyvcf2/helpers.o -W
no-sign-compare -Wno-unused-function -Wno-strict-prototypes -Wno-unused-result -Wno-discarded-qualifiers
      cyvcf2/helpers.c: In function ‘as_gts’:
      cyvcf2/helpers.c:7:21: warning: variable ‘found’ set but not used [-Wunused-but-set-variable]
           int missing= 0, found=0;
                           ^~~~~
      gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Ihtslib -Icyvcf2 -I/tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include -I/home/apps/python3/3.8.16/include/python3.8 -c htslib/bcf_sr_sort.c -o build/temp.linux-x86_64-cpython-38/htslib/bcf_sr_s
ort.o -Wno-sign-compare -Wno-unused-function -Wno-strict-prototypes -Wno-unused-result -Wno-discarded-qualifiers
      gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Ihtslib -Icyvcf2 -I/tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include -I/home/apps/python3/3.8.16/include/python3.8 -c htslib/bgzf.c -o build/temp.linux-x86_64-cpython-38/htslib/bgzf.o -Wno-sig
n-compare -Wno-unused-function -Wno-strict-prototypes -Wno-unused-result -Wno-discarded-qualifiers
      gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -Ihtslib -Icyvcf2 -I/tmp/pip-build-env-7wo204d6/overlay/lib/python3.8/site-packages/numpy/core/include -I/home/apps/python3/3.8.16/include/python3.8 -c htslib/cram/cram_codecs.c -o build/temp.linux-x86_64-cpython-38/htslib/cra
m/cram_codecs.o -Wno-sign-compare -Wno-unused-function -Wno-strict-prototypes -Wno-unused-result -Wno-discarded-qualifiers
      htslib/cram/cram_codecs.c:54:10: fatal error: ../htscodecs/htscodecs/varint.h: No such file or directory
       #include "../htscodecs/htscodecs/varint.h"
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/bin/gcc' failed with exit code 1
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for cyvcf2
Failed to build cyvcf2
ERROR: Could not build wheels for cyvcf2, which is required to install pyproject.toml-based projects]
```
Any hints on what went wrong and how to resolve are appreciated. 
Thanks!

**Comment by pur80a** [comment]
probably related to #270

**Comment by pur80a** [comment]
System info: 
```
$ uname -r
4.18.0-425.19.2.el8_7.x86_64
$ gcc --version | head -1
gcc (GCC) 8.5.0 20210514 (Red Hat 8.5.0-16.0.2)
```
Easily reproducible with: 
```
pip install cyvcf2==0.30.20
```

**Comment by brentp** [comment]
thanks for reporting and apologies for the delay.
as far as I can see, this is fixed with more recent versions such as 0.30.22.
